java/src/client/domestic/DomesticTradeController.java

```diff
@@ -492,6 +492,14 @@ public void unsetResource(ResourceType resource) {
 				receiveore = 0;
 				break;
 		}
+		if(sendbrick+sendwood+sendsheep+sendwheat+sendore <= 0) {
+			getTradeOverlay().setTradeEnabled(false);
+			getTradeOverlay().setStateMessage("Select Resource to Send to Other Player");
+		}
+		if(receivebrick+receivewood+receivesheep+receivewheat+receiveore <= 0) {
+			getTradeOverlay().setTradeEnabled(false);
+			getTradeOverlay().setStateMessage("Select Resource to Receive from Other Player");
+		}
 	}
```

Comment:
you can collapse these into a single if statement to not duplicate those 2 lines.